src/pages/view/styles/ItemGrid.module.css

```diff
@@ -4,7 +4,6 @@
 }
 .grid {
     margin-block-start: var(--spacers-dp8);
-    background-color: #f4f6f8;
```

Comment:
This was added as part of the slideshow. Have you double-checked that there's no interruption of the grid bg color when entering slideshow, or switching between items?

Reply:
FYI: this change was done by me, but due to the squash commit I need to go back to the original PR to work out when I did this. And hopefully work out why as well. If I can't figure it out then perhaps we should perhaps revert this grey200 back to `#f4f6f8` as well.

Reply:
Aha! It was a change made by @cooper-joe in https://github.com/dhis2/dashboard-app/commit/20513e046018f60825dfd2aaa092868924d3b7e0 so we need to keep it. And your background-color should then also be `var(--colors-grey200)`. Do you agree?

_I'll do a commit assuming that you do agree_

Reply:
I received confirmation from @cooper-joe. This PR is now good to go.